sentinel.conf

```diff
@@ -347,12 +347,12 @@ SENTINEL resolve-hostnames no
 #
 SENTINEL announce-hostnames no
 
-# When master_reboot_down_after_period is set to 0, Sentinel does not fail over
-# when receiving a -LOADING response from a master. This was the only supported
-# behavior before version 7.0.
+# When primary_reboot_down_after_period is set to 0, Sentinel does not fail over
```

Comment:
The config name is with dashes.

```suggestion
# When primary-reboot-down-after-period is set to 0, Sentinel does not fail over
```

Reply:
nice catch, I even do not find this for so long time, Thanks